test/integration.js

```diff
@@ -7,7 +7,8 @@ require('chai').should()
 
 describe('c8', () => {
   it('reports coverage for script that exits normally', () => {
-    const {output} = spawnSync(c8Path, [
+    const {output} = spawnSync('node', [
```

Comment:
It should use `process.execPath` instead of `node`.
As is, this would fail if `node` is not in the `PATH`. (It may also cause subtle bugs if there are multiple versions of Node)

Reply:
Thanks! fixed.